### DISCUSSION TITLE
Text styling with added fonts

### DISCUSSION THREAD

**Discussion by mchibok** [py-pdf/fpdf2#665] [General]
I needed to use a unicode font because other fonts included (times and helvetica) in the module could not print a long hyphen ("–").

So I tried a couple of other fonts, but everytime I try to use a particular [font style](https://pyfpdf.github.io/fpdf2/TextStyling.html), I get an "Undefined font" error:
- Roboto: Undefined font: robotoB
- DejaVuSans: Undefined font: dejavusansB

They were added with:
```py
pdf.add_font('DejaVuSans', '', fname='./font/DejaVuSans.ttf')
pdf.add_font('roboto', '', fname='./font/Roboto-Regular.ttf')
```

What is the best way to go: should I change the font names in the folder (Roboto-Bold.ttf to RobotoB.ttf and DejaVuSans-Bold.ttf to DejaVuSansB.ttf) for the module to pick them up automatically or should I instead add the bold font and rename it and use that name when I want to use the bold styling as in: `pdf.add_font('robotobold', '', fname='./font/Roboto-Bold.ttf')` ?

Thanks for your help!

**Comment by mchibok** [comment]
Well, I guess I didn't read the docs correctly and realised with the help of this [video](https://youtu.be/o7vixt2mVdo?t=240) that you can set bold and italics to the same font name by specifying 'style' parameter.
Thank you! :)

**Reply by Lucas-C** [reply]
Glad you found a solution by yourself! 😊
If you have any suggestion to improve some sections of our documentation about this, they would be welcome!
You can even submit a PR if you want.
